src/components/Table/Table.tsx

```diff
@@ -0,0 +1,10 @@
+import { Table } from "@mantine/core";
+import { FC } from "react";
+import TableModel from "./Table.types";
+import React from "react";
+
+const TableComponent: FC<TableModel> = (props: TableModel) => {
+  return <Table />;
+};
+
+export default TableComponent;
```

Comment:
avoid `export default`

no reason the use `FC` interface here.
we only use `FC` when the component accepts `children`.
and since react v18 the `FC` interface has changed and no longer add `children` by default, so there's no need to use it anymore.